src/coreclr/dlls/mscoree/coreclr/GenClrDebugResource.ps1

```diff
@@ -0,0 +1,36 @@
+param (
+    [string]
+    $dac,
+    [string]
+    $dbi,
+    [string]
+    $out
+)
+
+function Parse-Int {
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [string]$hexValue
+    )
+    return [System.Int32]::Parse($hexValue, [System.Globalization.NumberStyles]::HexNumber)
+}
+
+$clrDebugResource = [System.IO.BinaryWriter]::new([System.IO.File]::OpenWrite($out))
+
+try {
```

Comment:
could you please add a comment that shows the cpp equivalent that gets deserialized?